docs/guide/en/dropdown.md

```diff
@@ -20,57 +20,27 @@ To get started, instantiate the **Dropdown** widget and add items to it. Here is
 
 declare(strict_types=1);
 
+use Yiisoft\Bootstrap5\ButtonVariant;
 use Yiisoft\Bootstrap5\Dropdown;
 use Yiisoft\Bootstrap5\DropdownItem;
 ?>
 
-<?= Dropdown::widget()
-    ->togglerContent('Toggle Dropdown')
-    ->items(
-        DropdownItem::link('Action', '#'),
-        DropdownItem::link('Another action', '#'),
-        DropdownItem::divider(),
-        DropdownItem::link('Separated link', '#'),
-    )
+// @link https://getbootstrap.com/docs/5.3/components/dropdowns/#examples
+<?= 
+    Dropdown::widget()
```

Comment:
```suggestion
<?= Dropdown::widget()
```

Link to it at the very beginning of the guide page.

Reply:
done.